CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.1.2
```

Comment:
Remove this line. I don't think this is a patch release. Version number should be written just before a new release.